src/loaders/FileLoader.js

```diff
@@ -125,7 +125,7 @@ class FileLoader extends Loader {
 										for ( let i = 0, il = callbacks.length; i < il; i ++ ) {
 
 											const callback = callbacks[ i ];
-											if ( callback.onProgress ) callback.onProgress( event );
+											if ( callback.onProgress ) callback.onProgress( event, response.headers );
 
```

Comment:
I'm thinking instead of creating a `ProgressEvent` instance we just create a custom object with the information we want in the object. Something like:

```js
const event = {
  lengthComputable,
  loaded,
  total,
  response,
};
```

There's no reason the passed object has to be a `ProgressEvent`.

Reply:
Can it be `const event = new ProgressEvent( 'progress', { lengthComputable, loaded, total, response } );`?

Reply:
No from [the docs](https://developer.mozilla.org/en-US/docs/Web/API/ProgressEvent) "response" is not a valid field for ProgressEvent.

Reply:
Ah, yes... Then the object approach is better.

Reply:
That'd be great as well. Let me know if I should make the change, then I can change the docs accordingly.

Reply:
I'll have to defer to @mrdoob and @Mugen87 on whether or not the object approach is preferred.

Reply:
Maybe we can start with a custom object and if someone complains we fallback to the initial approach of the PR?

Reply:
I changed it to a custom object including the response and I also updated the docs.